### PR TITLE
Icon: Remove hardcoded color from `ai` icon

### PR DESCRIPTION
**Pull request by ashharrison90**
**What is this feature?**

- removes the hardcoded color from the `ai` icon and leaves it up to the consumer to implement
- there are 2 places this will affect in the grafana org:
  - `GenAIButton` these are all used within primary buttons anyway, so they get the color reapplied for free ✅ 
  - [`ExplanationButton`](https://github.com/grafana/machine-learning/blob/main/ui-plugins/grafana-ml-app/src/projects/Sift/Components/LLMPanel/ExplanationButton.tsx#L35) and [`Explanation`](https://github.com/grafana/machine-learning/blob/main/ui-plugins/grafana-ml-app/src/projects/Sift/Components/LLMPanel/Explanation.tsx#L44) in the ml app. this will change to white:
    |  |  |
    | --- | --- |
    | before | image |
    | after | image |
    - the workaround would be to pass the full icon component in to the button and specify the color, so:
    ```tsx
    // before
    <Button icon="ai" ... />
    
    // after
    <Button icon={<Icon name="ai" color={theme.colors.primary.main} />} ... />
    ```
    - have raised it with the ML team and they're happy to either leave it white or make the change 👍 
  
**Why do we need this feature?**

- so people can set the color of the ai icon correctly

**Who is this feature for?**

- everyone

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/img/icons/unicons/ai.svg [hunk 1]
```diff
@@ -1,27 +1,5 @@
...
-</linearGradient>
-<linearGradient id="paint2_linear_344_481" x1="6.5" y1="5.5" x2="15.124" y2="5.40875" gradientUnits="userSpaceOnUse">
-<stop stop-color="#6E9FFF" stop-opacity="0"/>
-<stop offset="0.164022" stop-color="#6E9FFF"/>
-<stop offset="0.197917" stop-color="#6E9FFF" stop-opacity="0.8075"/>
-<stop offset="0.426692" stop-color="#6E9FFF" stop-opacity="0"/>
-</linearGradient>
-</defs>
+<svg viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M37.3333 24C27.0252 24 18.6675 34.7456 18.6675 48C18.6675 34.7456 10.3098 24 0 24C10.3098 24 18.6675 13.256 18.6675 0C18.6675 13.256 27.0252 24 37.3333 24Z" fill="currentColor" fill-opacity="0.65"/>
```
Comment (joshhunt) [inline public/img/icons/unicons/ai.svg:2]: Should the shapes still be partly transparent?

Comment (ashharrison90) [reply public/img/icons/unicons/ai.svg:2]: good question - this is the icon i was given, so i guess not? 🤔 

cc @staton-hyse11

Comment (staton-hyse11) [reply public/img/icons/unicons/ai.svg:2]: It would be good to unify with the rest of our icons and have it solid - no transparencies

**Comment by ashharrison90** [comment]
@Clarity-89 do i also have to update this icon in the `@grafana/icons` package? 🤔

**Comment by Clarity-89** [comment]
> @Clarity-89 do i also have to update this icon in the `@grafana/icons` package? 🤔 

You can replace the file in the package's svg folder with the updated svg and it'd be enough.

**Review by joshhunt — APPROVED** [review]

**Review by Clarity-89 — APPROVED** [review]

**Pull request merged**